test/entity-test.js

```diff
@@ -486,6 +486,51 @@ describe('Entity', () => {
 
             expect(model.getEntityDataWithVirtuals.called).equal(true);
         });
+
+        it('should clear embedded object excluded properties', () => {
+            schema = new Schema({
+                embedded: { excludeFromRead: ['prop1', 'prop2'] },
+            });
+
+            ModelInstance = gstore.model('HasEmbedded', schema);
+
+            const entity = new ModelInstance({ embedded: { prop1: '1', prop2: '2', prop3: '3' } });
+            const plain = entity.plain({});
+
+            expect(typeof plain.embedded.prop1).equal('undefined');
```

Comment:
For consistency with other tests, could you change the check for undefined with `assert.isUndefined(...)`? thanks :+1: